src/fastlane/initFastlane.ts

```diff
@@ -0,0 +1,89 @@
+import { getPublicOrderId, getEnvironment, getShopIdentifier, getJwtToken } from '@boldcommerce/checkout-frontend-library';
+import { loadScript } from '@paypal/paypal-js';
+import {
+    loadJS,
+    getBraintreeJsUrls,
+    braintreeOnLoadClient,
+    IFastlaneInstance,
+    setFastlaneInstance,
+    getBraintreeClient,
+    IBraintreeClient,
+    FastlaneLoadingError,
+} from 'src';
+
+interface TokenResponse {
+    is_test_mode: boolean;
+    client_token: string;
+}
+
+interface BraintreeTokenResponse extends TokenResponse {
+    type: 'braintree';
+    client_id: null;
+}
+
+interface PPCPTokenResponse extends TokenResponse {
+    type: 'ppcp';
+    client_id: string;
+}
+
+export async function initFastlane(): Promise<void>  {
+    const {clientJsURL, dataCollectorJsURL, fastlaneJsURL} = getBraintreeJsUrls('3.101.0-fastlane-beta.7.2');
+
+    try {
+        const env = getEnvironment();
+        const shopId = getShopIdentifier();
+        const publicOrderId = getPublicOrderId();
+        const jwt = getJwtToken();
+        const resp = await fetch(`${env.url}/checkout/storefront/${shopId}/${publicOrderId}/paypal_fastlane/client_token`, {
+            headers: {
+                Authorization: `Bearer ${jwt}`,
+            },
+        });
```

Comment:
Please add a TODO and create a ticket to implement this API in Bold Frontend library.

Bold Frontend library has mechanisms to control retry on specifics failures, wrapps tha fetch on try catch to handle network call failures and is the centralize place to update changes on the Bold APIs.

Reply:
Added